### PR TITLE
deployment process refine

### PR DESCRIPTION
**Pull request by suiguoxin**
Fix part of the deployment issues listed in #5001:
- move requirement check before configuration setting;
- move quick-start-config folder setting to quick-start-services.sh; 
- fix installation document branch checkout issue; change suggested version to v1.3.y;
- refine uninstallation doc
- fix git clone issue: clone to an existing folder may raise error, so the code pai/kubespray may be unsuccessfully cloned.

### CODE REVIEW

File: docs/manual/cluster-admin/installation-guide.md [hunk 1]
```diff
@@ -192,8 +192,7 @@ docker_image_tag: v1.0.0
 On the dev box machine, use the following commands to clone the OpenPAI repo:
 
 ```bash
-git clone https://github.com/microsoft/pai.git
-git checkout pai-1.0.y  # change to a different branch if you want to deploy a different version
+git clone -b pai-1.3.0 https://github.com/microsoft/pai.git # change to a different branch if you want to deploy a different version
```
Comment (hzy46) [inline docs/manual/cluster-admin/installation-guide.md:195]: pai-1.3.y

File: docs_zh_CN/manual/cluster-admin/installation-guide.md [hunk 2]
```diff
@@ -191,8 +191,7 @@ docker_image_tag: v1.0.0
 在dev box机器上，使用下面的命令克隆OpenPAI的源代码。
 
 ```bash
-git clone https://github.com/microsoft/pai.git
-git checkout pai-1.0.y  # 如果您想要部署不同的版本，请切换到相应的branch。
+git clone -b pai-1.3.0 https://github.com/microsoft/pai.git # 如果您想要部署不同的版本，请切换到相应的branch。
```
Comment (hzy46) [inline docs_zh_CN/manual/cluster-admin/installation-guide.md:194]: pai-1.3.y

File: docs/manual/cluster-admin/installation-guide.md (hunk 1 above)
Comment (suiguoxin) [reply docs/manual/cluster-admin/installation-guide.md]: > pai-1.3.y

Fixed, thanks

File: docs_zh_CN/manual/cluster-admin/installation-guide.md (hunk 2 above)
Comment (suiguoxin) [reply docs_zh_CN/manual/cluster-admin/installation-guide.md]: > pai-1.3.y

Fixed, thanks

File: docs/manual/cluster-admin/how-to-uninstall-openpai.md [hunk 3]
```diff
@@ -2,10 +2,10 @@
 
 The uninstallation of OpenPAI is irreversible: all the data will be removed and you cannot find them back. If you need a backup, do it before uninstallation.
 
-First, log in to the dev box machine and delete all PAI services:
+First, log in to the dev box machine and delete all PAI services with dev box:
```
Comment (hzy46) [inline docs/manual/cluster-admin/how-to-uninstall-openpai.md:5]:
```
with [dev box container](./basic-management-operations.md#pai-service-management-and-paictl).
```

**Review by hzy46 — APPROVED** [review]

**Pull request merged**